content/vi/docs/reference/kubectl/cheatsheet.md

```diff
@@ -0,0 +1,380 @@
+KUBECONFIG=~/.kube/config:~/.kube/kubconfig2 
+
+kubectl config view
+
+# lấy mật khẩu cho người dùng e2e
+kubectl config view -o jsonpath='{.users[?(@.name == "e2e")].user.password}'
+
+kubectl config view -o jsonpath='{.users[].name}'    # hiển thị người dùng đầu tiên  
+kubectl config view -o jsonpath='{.users[*].name}'   # lấy danh sách người dùng  
+kubectl config get-contexts                          # hiển thị danh sách các ngữ cảnh 
+kubectl config current-context                       # hiển thị ngữ cảnh hiện tại
+kubectl config use-context my-cluster-name           # thiết lập ngữ cảnh mặc định cho my-cluster-name
+
+# thêm một cụm mới vào kubeconf hỗ trợ xác thực cơ bản
+kubectl config set-credentials kubeuser/foo.kubernetes.com --username=kubeuser --password=kubepassword
+
+# lưu vĩnh viễn namespace cho tất cả các lệnh kubectl tiếp theo trong ngữ cảnh đó
+kubectl config set-context --current --namespace=ggckad-s2
+
+# thiết lập ngữ cảnh sử dụng tên người dùng và namespace cụ thể
+kubectl config set-context gce --user=cluster-admin --namespace=foo \
+  && kubectl config use-context gce
+ 
+kubectl config unset users.foo                       # xóa người dùng foo
+```
+
+## Apply
+`apply` quản lý các ứng dụng thông qua các tệp định nghĩa tài nguyên Kubernetes. Nó tạo và cập nhật các tài nguyên trong một cụm thông qua việc chạy `kubectl apply`. Đây là cách được đề xuất để quản lý các ứng dụng Kubernetes trong thực tế. Xem thêm [Kubectl Book](https://kubectl.docs.kubernetes.io).
+
+## Tạo một đối tượng
+
+Kubernetes manifests có thể được định nghĩa trong tệp json hoặc yaml. Phần mở rộng `.yaml`,
+`.yml`, và `.json` có thể được dùng.
+
+```bash
+kubectl apply -f ./my-manifest.yaml            # tạo tài nguyên
+kubectl apply -f ./my1.yaml -f ./my2.yaml      # tạo từ nhiều tệp
+kubectl apply -f ./dir                         # tạo tài nguyên từ tất cả các tệp manifest trong thư mục dir
+kubectl apply -f https://git.io/vPieo          # tạo tài nguyên từ url
+kubectl create deployment nginx --image=nginx  # tạo một deployment nginx
+kubectl explain pods,svc                       # lấy thông tin pod và service manifest
+
+# Tạo nhiều đối tượng YAML từ stdin
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: busybox-sleep
+spec:
+  containers:
+  - name: busybox
+    image: busybox
+    args:
+    - sleep
+    - "1000000"
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: busybox-sleep-less
+spec:
+  containers:
+  - name: busybox
+    image: busybox
+    args:
+    - sleep
+    - "1000"
+EOF
+
+# Tạo một secret với một số keys
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+type: Opaque
+data:
+  password: $(echo -n "s33msi4" | base64 -w0)
+  username: $(echo -n "jane" | base64 -w0)
+EOF
+
+```
+
+## Xem, tìm các tài nguyên
+
+```bash
+# Lệnh get với một số đầu ra cơ bản
+kubectl get services                          # Liệt kê tất cả các dịch vụ trong namespace

```

Comment:
we shouldn't translate primitives to Vietnamese. Just call it `services`

Reply:
I've just updated the document to resolve your change request.